apifuzzer/swagger_template_generator.py

```diff
@@ -48,9 +48,20 @@ def process_api_resources(self):
                                 name=param['name'],
                                 value=get_sample_data_by_type(param.get('type'))
                             ))
-                    else:
+                    elif param.get('in') == ParamTypes.QUERY:
                         fuzz_type = get_fuzz_type_by_param_type(param.get('type'))
-                        template.data = fuzz_type(name=param['name'], value=get_sample_data_by_type(param.get('type')))
+                        template.parameters.append(
+                            fuzz_type(
+                                name=param['name'],
+                                value=get_sample_data_by_type(param.get('type'))
+                            ))
+                    elif param.get('in') == ParamTypes.BODY:
+                        fuzz_type = get_fuzz_type_by_param_type(param.get('type'))
+                        template.data = fuzz_type(
+                            name=param['name'],
+                            value=get_sample_data_by_type(param.get('type'))
+                        )
+                    # TODO implement FORM_DATA, COOKIE
```

Comment:
form data should be sent in body and cookie should be sent in header, isn't it?
Please do not remove else: at least put some logging there.

Reply:
Agree. It will be fixed.

Reply:
Form data and cookies parsing have been fixed. I put a logger statement in an else block but it doesn't work. I have no idea why.